packages/flutter_tools/templates/plugin/lib/projectName.dart.tmpl

```diff
@@ -1,19 +1,16 @@
 {{#no_platforms}}
-// You have generated a new plugin project without
-// specifying the `--platforms` flag. A plugin project supports no platforms is generated.
-// To add platforms, run `flutter create -t plugin --platforms <platforms> .` under the same
-// directory. You can also find a detailed instruction on how to add platforms in the `pubspec.yaml` at https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms.
+// You have generated a new plugin project without specifying the `--platforms`
+// flag. A plugin project supports no platforms is generated. To add platforms,
```

Comment:
This sentence doesn't actually make sense (a pre-existing issue); as long as you're adjusting the comment let's fix it. I think it was intended to say something like "A plugin project that supports no platform was generated."